src/sass/partials/components/_nav.scss

```diff
@@ -41,3 +41,47 @@
     }
   }
 }
+
+.breadcrumb {
+
+  &--list {
+    background-color: transparent;
+    display: flex;
+    margin-top: 0;
+    transition: .15s transform ease-out;
```

Comment:
The transitions [we already have](https://github.com/punchcard-cms/punchcard/blob/master/src/sass/partials/global/_variables.scss) are all pegged at `.2s`. Why is this `.15s`?